images/capi/ansible/windows/roles/systemprep/tasks/main.yml

```
@@ -33,6 +33,9 @@
   win_updates:
     whitelist: "{{ windows_updates_kbs_numbers }}"
     reboot: yes
+    category_names:
+      - Updates
+      - Security Updates
```

```suggestion
    category_names:
      - Updates
      - UpdateRollups
      - SecurityUpdates
```

I'm not sure what categories are included in Updates but I would expect the most popular KBs for this will be the cumulative updates so maybe including rollups will be good here?

@jsturtevant do you know of a reference that tells us what is included in each category?

---

https://docs.microsoft.com/en-us/troubleshoot/windows-client/deployment/standard-terminology-software-updates